src/VolumeDrawable.ts

```diff
@@ -905,6 +905,10 @@ export default class VolumeDrawable {
     return this.childObjects.has(line);
   }
 
+  hasLineObjects(): boolean {
+    return this.childObjects.size > 0;
```

Comment:
Does `childObjects` ever have anything other than lines in it? I'm seeing isosurface meshes get put in there too I think

Reply:
Good question, and yeah, it would contain the meshes. I'm going to actually just remove this API because I don't think I ended up needing to call it anywhere.